src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/columnDefinitions/publicId.tsx

```diff
@@ -0,0 +1,24 @@
+import { ColumnDef } from "@tanstack/react-table";
+import { generateWidthStyles } from "src/common/utils/tableUtils";
+import { SortableHeader } from "src/views/Data/components/SortableHeader";
+import DefaultCell from "../components/DefaultCell";
+
+export const publicIdColumn: ColumnDef<Sample, any> = {
+  id: "publicId",
+  accessorKey: "publicId",
+  header: ({ header, column }) => (
+    <SortableHeader
+      header={header}
+      style={generateWidthStyles(column)}
+      tooltipStrings={{
+        boldText: "Public ID",
+        regularText:
+          "This is your GISAID ID or public ID generated by CZ Gen Epi.",
```

Comment:
This text is different for mpox:
"This is your GenBank Accession or public ID generated by CZ GEN EPI.

Reply:
Everything else in this column is the same, I'm not sure if we should add a strings file for this or create a separate column file...